### DISCUSSION TITLE
root ERROR Cannot find a service for the path: /services/glsp/WorkflowNotation/:id

### DISCUSSION THREAD

**Discussion by shinesamjacob** [eclipse-glsp/glsp#629] [General]
I am developing glsp application, i am getting the above error when I try to launch the application. If I try restarting the server multiple times, it works without any problem. Any ideas on what could be the problem?.( I have modified the existing coffee editor example, so the contribution id is "WorkflowNotation" ). Is it related to backend or frontend. When I try debug the front end code, it is failing at websocket connection.

**Comment by planger** [comment]
This looks like https://github.com/eclipse-glsp/glsp/discussions/558.
Strange however that it works when restarting multiple times. Could be a timing issue that the server takes a long time to come up? Does it also work if you start the server, wait a bit and then refresh the browser (even multiple times)?

Is there any notable difference between your code and the [workflow example's Theia integration](https://github.com/eclipse-glsp/glsp-theia-integration/tree/master/examples/workflow-theia).

**Comment by shinesamjacob** [comment]
Theia integration example and coffee editor example have little difference. But I am able to reproduce the same issue in the coffee editor also.
One change I did on the coffee editor was, removed all generator, DSL and Analyser extensions and kept only GLSP extension. Also did not modify the background at all.

**Comment by shinesamjacob** [comment]
![image](https://user-images.githubusercontent.com/19471062/165721497-44206729-e264-4c28-81c1-08be40beb452.png)

**Comment by mishaltripathi** [comment]
Hi, I tried to build and run the coffee editor example. I too encountered with the same error(_Cannot find a service for the path: /services/glsp/WorkflowNotation/:id_) Please look at the screenshot. Please let me know if someone finds root cause or solution. Thanks!!

![Error](https://user-images.githubusercontent.com/45622489/165745343-c388fbd3-8e68-4b03-9d22-9b54dbbdaebe.PNG)

Note: I didn't modify or touch the coffee editor code.
